### DISCUSSION TITLE
Unable to disable in VS2022

### DISCUSSION THREAD

**Discussion by JohnathanKG** [community/community#51899] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

**Description**: Copilot suggestion persists after disabling, restarting Visual Studio
**Steps to reproduce**: Make a C# project, trigger an inline suggestion, disable Copilot for the solution and the suggestion persists even after restarting Visual Studio.
**Expected behavior**: Inline suggestion should not persist after disabling Copilot, and restarting Visual Studio

**Comment by ayim** [comment, marked as answer]
Are you sure these aren't IntelliCode's whole line completions?
If you disable Copilot and its cloud model, we fallback to using the IntelliCode local model.
Key difference being that the IntelliCode model will only ever give you up to a line of grey text at once.

**Reply by JohnathanKG** [reply]
Thank you for clarifying, that was the issue. I apologize for the confusion. Thanks!
